package Exception;

public class Text {

		public static void main(String[] args) {
	
			System.out.println("Connection 1 is created vijay vijay");
			int a = 10;
			int b=10;
			try
			{
			int c=a/b;
			System.out.println(c);
			int arr[] = new int[5];
			arr[10] = 77;
			System.out.println(arr[1]);
			}
			catch(Exception e)
			{
				System.out.println("I don't know what is this excepiton "+e);
				
			}
			System.out.println("Connection 1 is terminated vijay");
	} 

}
